README.md

```diff
@@ -1,15 +1,193 @@
 
 # Library Management System
 
-This is a Library Management System built with Spring Boot.
+This is a Spring Boot application that implements books, authors, users, tasks etc. that are gathered in a MySQL database. Each role has a different degree of access to the application.
+ 
 
-### Start application
+### How do I start the application?
```

Comment:
Use statements rather than questions for headings and sub-headings. The latter is less formal.
Here is an example of README file from Baeldung https://github.com/eugenp/tutorials/tree/master/spring-web-modules/spring-thymeleaf.